sources/us/ms/madison.json

```diff
@@ -14,6 +14,7 @@
     "conform": {
         "street": "PHYSTREET",
         "number": "PHYNUM",
-        "type": "geojson"
+        "type": "geojson",
+        "postcode": "zip"
```

Comment:
us/ms/madison is currently a broken source (the layer doesn't exist), so I'm not sure where `zip` is coming from here.  However, I think the relocated parcels source is here:

http://gis.cmpdd.org/arcgis/rest/services/County/Madison_County/MapServer/11

Feel free to update to this source using `STREET_NUMBER ` and `STREET_NAME_1 ` for the number and street (there are city/state/zip fields but they refer to the owners' mailing address).